FogCreek/Glitch-Community
# Gaudy report: hookify current user

## Links
* Remix link: https://gaudy-report.glitch.me/

## Changes:
* hookify CurrentUser container
* also, hookify Popover container -- now there are no more components to hookify 

## How To Test:
* open two tabs with https://gaudy-report.glitch.me/
* in one tab, login with email + code
* your account should load in both windows
* logout in one tab
* your account should unload in both windows
